### PR TITLE
September - API Improvements and DNS Domains

### PR DESCRIPTION
**Pull request by Jump-Suit**
VeeMee - Added initial support for the Wardrobe Wars minigame
Hellfire Games - Further added support for Home Tycoon game, not final.
PremiumAgency - Added initial support for Fifa Sim Predictor, adding the Accounts module to the API. Added Halloween 2010 support. Resources.cs can now determine what XML localise to choose from based on POST.
MITMDNS - Added tons of domains and reorganized its layout for better clarity.

Begun tweaking MUISClass.cs to have the ORIGINAL DOMAINS FOR MAS AND SVO. MultiServer should redirect these automatically in MITMDNS, why are we using the IP variable... when MITMDNS can take care of it?...

### CODE REVIEW

**Comment by GitHubProUser67** [comment]
It's really typo, but can you update the PR name to reflect the changes? I am trying to be more accurate with my commits too.

File: BackendServices/AuxiliaryServices/WebAPIService/VEEMEE/VEEMEEClass.cs [hunk 1]
```diff
@@ -27,6 +28,13 @@ public VEEMEEClass(string method, string absolutepath)
             string result = null;
             string resultContentType = null;
 
+            if(absolutepath.Contains("//WardrobeWars/Images"))
```
Comment (GitHubProUser67) [inline BackendServices/AuxiliaryServices/WebAPIService/VEEMEE/VEEMEEClass.cs:31]: Can you add a method check? Like GET.

File: WebServers/MitmDNS/static/routes.txt [hunk 2]
```diff
@@ -23,111 +23,347 @@
...
+
+;PS3 Misc
+starhawk.telemetry.online.scea.com,Redirect,IP
+singstar.telemetry.online.scea.com,Redirect,IP
+api.singstar.online.scee.com,Redirect,IP
+
+;Quazal
+tdulive.quazal.net,Redirect,IP
+ridingclubpclive.quazal.net,Redirect,IP
+orb.testdriveunlimited.com,Redirect,IP
```
Comment (GitHubProUser67) [inline WebServers/MitmDNS/static/routes.txt:340]: This one is not quazal, that was the trap ^^. It's a Eden dns domain.

File: WebServers/MitmDNS/static/routes.txt [hunk 3]
```diff
@@ -23,111 +23,347 @@
...
+turoklive.quazal.net,Redirect,IP
+ghostps3live.quazal.net,Redirect,IP
+master10.doublefusion.com,Redirect,IP
```
Comment (GitHubProUser67) [inline WebServers/MitmDNS/static/routes.txt:343]: Is this quazal?

**Comment by GitHubProUser67** [comment]
The review is done, little to change in order for it to be merged, good job on this!

File: WebServers/MitmDNS/static/routes.txt (hunk 3 above)
Comment (GitHubProUser67) [reply WebServers/MitmDNS/static/routes.txt:343]: PS: it is a ubisoft domain.

File: BackendServices/AuxiliaryServices/WebAPIService/VEEMEE/VEEMEEClass.cs (hunk 1 above)
Comment (Jump-Suit) [reply BackendServices/AuxiliaryServices/WebAPIService/VEEMEE/VEEMEEClass.cs:31]: I don't see why for a specific endpoint that ONLY WILL EVER PULL IMAGES... but fine.

**Comment by Jump-Suit** [comment]
user.cs in OHS 
image
This is only used in the SCEAsia Sodium home square.

**Pull request merged**